jina/peapods/zmq/__init__.py

```diff
@@ -231,22 +240,73 @@ def print_stats(self):
             f'recv_size: {get_readable_size(self.bytes_recv)}'
         )
 
+    def _init_dynamic_out_socket(self, host_out, port_out):
+        out_sock, _ = _init_socket(
+            self.ctx,
+            host_out,
+            port_out,
+            SocketType.DEALER_CONNECT,
+            self.identity,
+            ssh_server=self.args.ssh_server,
+            ssh_keyfile=self.args.ssh_keyfile,
+            ssh_password=self.args.ssh_password,
+        )
+        self.logger.debug(
+            f'output {self.args.host_out}:{colored(self.args.port_out, "yellow")}'
+        )
+        return out_sock
+
+    def _get_dynamic_out_socket(self, target_pod, as_streaming=False):
```

Comment:
what is this `as_streaming` and where is it used?

Reply:
It is used in the `ZmqStreamlet`.